pkgs/desktops/kde-5/applications/kmix.nix

```diff
@@ -0,0 +1,30 @@
+{
+  kdeApp, lib, kdeWrapper,
+  ecm, kdoctools,
+  kglobalaccel, kxmlgui, kcoreaddons, kdelibs4support,
+  plasma-framework
+}:
+
+let
+  unwrapped =
+    kdeApp {
+      name = "kmix";
+      meta = {
+        license = with lib.licenses; [ gpl2 lgpl21 fdl12 ];
+        #maintainers = [ lib.maintainers.ttuegel ];
```

Comment:
I am trying to add myself as maintainer... The instruction in `default.nix` of kde-5/applications doesn't make sense at all.